plugins/catalog-backend/seeds/b_entities.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('entities').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('entities').insert([
+        {
+          id: 14, 
+          location_id: 1,
+          etag: 'abc',
+          generation: 0,
+          api_version: '0.1',
+          kind: 'Component',
+          name: 'Component 1',
+          namespace: '',
```

Comment:
namespaces are either null or a valid string, the default is the string `default`